README.md

```diff
@@ -144,6 +144,12 @@ With Chocolatey installed, execute the following command to install posh-git:
 choco install poshgit
 ```
 
+### Installing post-git Manually
+
+If you need to test/debug changes prior to contrubiting here, or would otherwise prefer to install post-git without the aid of a package manager you can run `Import-Module POST-GET-PATH` where `POST-GET-PATH` is the path to the `src\posh-git.psd1` file.
```

Comment:
There's a typo on this line - `contrubiting` should be `contributing`.  Also rather than say "run" it might be better to say "execute".